docs/timezone.md

```diff
@@ -135,9 +217,7 @@ tz2 = Temporal.TimeZone.from(tz);
 ### timeZone.**id** : string
 
 The `id` property gives an unambiguous identifier for the time zone.
-Effectively, this is the canonicalized version of whatever `timeZoneIdentifier` was passed as a parameter to the constructor.
-
-When subclassing `Temporal.TimeZone`, this property doesn't need to be overridden because the default implementation gives the result of calling `toString()`.
+Effectively, this is the [canonicalized](#canonicalization-of-time-zone-identifiers) version of whatever identifier was provided when the `Temporal.TimeZone` instance was created.
```

Comment:
This is no longer canonicalized, right?

Reply:
Yes. The current docs have more currently-accurate text here. This change will be removed in my next rebase.